solidity/interfaces/IOngoingAirdrops.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+/**
+ * @title A contract that will enable users to claim from multiple ongoing airdrops, and each airdorp will have a deadline.
+ * @notice This contract will enable multiple ongoing and updateable airdrops (called campaigns). Each of this campaigns
+ *         will have an updateable deadline, and will be able to hold airdrop information for multiple tokens.
+ *         Additionally, governor will be able to shutdown a campaign whenever it choses to. Since this is a pretty
+ *         permissioned contract, governor has full control over funds.
+ */
+
+interface IOngoingAirdrops {
+  /// @notice Token and amount information
+  /// @dev Struct that will hold an ERC20 token, and an amount.
+  struct TokenAmount {
+    IERC20 token;
+    uint256 amount;
+  }
+
+  /// @notice Thrown when a campaign is invalid, or zero bytes.
+  error InvalidCampaign();
```

Comment:
It's hard to understand if this make sense or not without the code, but we can remove them later if we realized that they don't make sense